src/widget/canvas.rs

```diff
@@ -1,29 +1,6 @@
-use {
-    Align,
-    CharacterCache,
-    Color,
-    Colorable,
-    Dimensions,
-    FontSize,
-    Frameable,
-    FramedRectangle,
-    IndexSlot,
-    Labelable,
-    Padding,
-    Place,
-    Position,
-    Positionable,
-    Range,
-    Rect,
-    Scalar,
-    Sizeable,
-    TextWrap,
-    Theme,
-    TitleBar,
-    Ui,
-    UiCell,
-    Widget,
-};
+use {Align, CharacterCache, Color, Colorable, Dimensions, FontSize, Frameable, FramedRectangle,
+     IndexSlot, Labelable, Padding, Place, Position, Positionable, Range, Rect, Scalar, Sizeable,
+     TextWrap, Theme, TitleBar, Ui, UiCell, Widget};
```

Comment:
I'm a little surprised by these changes - I would have thought having each name on their own line would be better for refactoring (maybe that's just the case for my vim editing though).